mscore/plugin/api/elements.h

```diff
@@ -348,6 +352,11 @@ class Note : public Element {
       Q_PROPERTY(QQmlListProperty<Ms::PluginAPI::Element>  dots              READ dots)
 //       Q_PROPERTY(int                            dotsCount         READ qmlDotsCount)
       Q_PROPERTY(QQmlListProperty<Ms::PluginAPI::Element>  elements          READ elements)
+      /// List of PlayEvents associated with this note.
+      /// Important: You must call Score.createPlayEvents()
+      /// to see meaningfull data in the PlayEvent lists.
+      /// \since MuseScore 3.4
```

Comment:
3.4? Next version will be 3.3 IIRC

Reply:
I figured the 3.3 release was imminent and I wanted more testing on the feature before it goes it.

-Dale

Reply:
I don't see any signs of 3.3 being imminent. @anatoly-os ?

Reply:
Well it's all set back to 3.3 now since things are going well with the PR.